mscore/preferences.cpp

```diff
@@ -1154,13 +1154,13 @@ void PreferenceDialog::resetShortcutClicked()
     
 void PreferenceDialog::saveShortcutListClicked()
       {
-      QString saveFileName = QFileDialog::getSaveFileName(this, tr("Save Shortcuts"), "", tr("MuseScore Shortcuts File") + " (*.xml)");
+      QString saveFileName = QFileDialog::getSaveFileName(this, tr("Save Shortcuts"), "", tr("MuseScore Shortcuts File") + " (*.xml)", 0, preferences.nativeDialogs ? QFileDialog::Options() : QFileDialog::DontUseNativeDialog);
```

Comment:
I tried this. The other dialogs appear to be the same and it seems to be working fine.